### ISSUE TITLE
Committing segfaults if there is no duet set up

### ISSUE THREAD

**Issue by tomwhoiscontrary** [git-duet/git-duet#8]
I have a fairly new project which is, as far as i know, pretty normal. I have been using the Go version of git duet quite happily. I somehow lost my selected duet; `git duet` produced no output. Before i realised this, i attempted to commit, and:

```
$ git duet-commit -m "Script build process"
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x2f1a]

goroutine 1 [running]:
main.main()
    /home/travis/gopath/src/github.com/hamfist/go-git-duet/src/github.com/jszwedko/go-git-duet/git-duet-commit/main.go:46 +0xf1a
```

Setting up the duet fixed this.

**Comment by jszwedko** [comment]
Put #13 together for this, thanks for reporting @tomwhoiscontrary !

**Comment by jszwedko** [comment]
Merged #13, but let me know if you have any comments!
